hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -356,18 +348,27 @@ private void handleEvents(
                 }
             }
         }
+        final boolean isGenesis =
```

Comment:
Just a slight performance optimization, is it worth keeping this as a class field so we don't have to keep recomputing it? Seems like it'll be true just once, then always false

Reply:
yeah true .. agreed.